docs/mobility/slam/cartographer/_index.md

```diff
@@ -18,6 +18,9 @@ aliases:
 To use Cartographer with the Viam {{< glossary_tooltip term_id="slam" text="SLAM" >}} service, you can use the [`cartographer`](https://app.viam.com/module/viam/cartographer) {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}}.
 See the [Use Modules](/registry/#use-modules) section for instructions on using a module from the Viam registry on your machine.
 
+This module utilizes the Cartographer SLAM algorithm to provide tools for generating SLAM maps on your machine.
+If you want to use a SLAM algorithm outside of Cartographer, you can modify this module or create a new one to suit your needs.
```

Comment:
```suggestion
If you want to use a SLAM algorithm outside of Cartographer, you can create a new module to suit your needs.
```

Suggesting this because we probably wouldn't want users to modify the cartographer module we wrote, but they could certainly create a new one